app/src/containers/b2b/AccountMain.tsx

```diff
@@ -423,6 +441,30 @@ export default class AccountMain extends React.Component<RouteComponentProps<Acc
                 </table>
               </div>
 
+              <Modal
+                open={isDeleteAssociateOpen}
+                onClose={this.handleDeleteModalClose}
+                classNames={{ modal: 'b2b-delete-associate-dialog', closeButton: 'b2b-dialog-close-btn' }}
+              >
+                <div className="dialog-header">{intl.get('remove-associate')}</div>
+                <div className="dialog-content">
+                  <p>
+                    {intl.get('confirm-delete-associate')}
+                  </p>
+                </div>
+                <div className="dialog-footer">
+                  <button className="cancel" type="button" onClick={this.handleDeleteModalClose}>{intl.get('no')}</button>
+                  <button className="upload" type="button" onClick={() => this.handleDeleteAssociateClicked(associateUri)}>
+                    {intl.get('yes')}
+                  </button>
+                </div>
+                {isLoading ? (
```

Comment:
I don't see this Loader being used or triggered in the modal. Can we remove this?

Reply:
Nvm, we'll keep in case the request actually hangs as an error case 😄